app/frontend/src/components/HorizontalScroller.tsx

```diff
@@ -8,23 +8,29 @@ import useOnVisibleEffect from "../utils/useOnVisibleEffect";
 import CircularProgress from "./CircularProgress";
 
 const useStyles = makeStyles((theme) => ({
-  padder: {
-    height: 1,
-    width: 0.1,
-  },
   root: {
     "& > *": {
-      flex: "0 0 auto",
-    },
-    "& > * + *": {
-      marginInlineStart: theme.spacing(2),
+      flexShrink: 0,
+      height: "calc(100% - 2rem)",
+      width: "85vw",
+      maxWidth: "33rem",
+      marginRight: theme.spacing(2),
+      marginBottom: 0,
+      scrollSnapAlign: "start",
     },
     alignItems: "center",
-    display: "flex",
+    display: "inline-flex",
     flexDirection: "row",
-    overflowX: "auto",
-    padding: theme.spacing(2),
+    height: "100%",
+    width: "100%",
+    padding: theme.spacing(0, 2),
     WebkitOverflowScrolling: "touch",
+    overflowX: "scroll",
+    scrollSnapType: "x mandatory",
+    scrollPadding: theme.spacing(1.5),
```

Comment:
I think we probably want to avoid using `0.5` units in the spacing value, so we should go with 1 or 2? Otherwise we are just moving off the 8px scale we agreed with design (even though I know they haven't been the best at sticking to it in the Figma designs) and defeats the point of having one.

Reply:
I totally agree to follow the grid, but in some cases, the 4px unit will be the only way to do that (for example if you apply margin-left and margin-right to a set of elements in order to achieve an 8px gutter while still having them centered. Here it is more a visual matter. If we apply scroll-padding 16px, the box-shadow of the previous card enters in the screen (in the left part of the scroller), which is not intended.
![Captura de pantalla de 2021-06-06 13-01-43](https://user-images.githubusercontent.com/8875832/120922228-2e735000-c6c8-11eb-816a-15e956a19897.png)

But if we apply scroll-padding: 12px, we get rid of that piece of shadow, and the proportion is still nice (3:4, which is used all over the design).
![Captura de pantalla de 2021-06-06 13-01-21](https://user-images.githubusercontent.com/8875832/120922148-cf154000-c6c7-11eb-99d7-8879d459da7e.png)

Reply:
Hmm maybe we should review the scale to be `4px` based instead. I'm seeing more and more cases where we are just breaking out of it, which suggests to me it's not granular enough or designs need to adjust to having more spaces around things...

Reply:
I think allow half in special circumstances. Having 4px grid would mean we would have lots of `theme.spacing(4)` which seems too high for the most common spacing.